### DISCUSSION TITLE
Extruded solid polygon wireframe - custom wireframe

### DISCUSSION THREAD

**Discussion by MartinP-C** [visgl/deck.gl#7157] [Q&A]
As I understand it from the deck.gl docs, the `wireframe: true` option will only allow a 1px line.
I want to create a more obvious wireframe, probably starting with just a thicker line.

The docs mention that two layers should be used. I'm assuming one for the solid polygon, and another for the custom wireframe
```Wireframe and solid extrusions are exclusive, you'll need to create two layers with the same data if you want a combined rendering effect.```

Can somebody point me in the right direction for which layer can be used to create the custom wireframe?
I can't see any that would allow drawing a thicker line.

**Comment by Pessimistress** [comment]
Use `PathLayer` with `billboard: true`.

**Reply by MartinP-C** [reply]
Should that also render vertical lines for the vertices?
Using a `PathLayer` with `billboard: true` and any combination of width only renders a polygon outline on the base of the extrusion.
